src/components/cart/cartProduct.js

```diff
@@ -0,0 +1,64 @@
+import './index.css'
+import {useState} from 'react'
+import {Button} from '../index'
+import {useCart} from '../../contexts/cartContext'
+import { GoX } from "react-icons/go";
+import axios  from "axios";
+
+const CartProduct=({product})=>{
+  const {dispatchCart,cartCount}=useCart()
+  const {title,imageUrl,price}=product
+  const[quantityPrice,setQuantityPrice]=useState(Number(price))
+  const [count,setCount]=useState(1)
+  const isdisable=count===1?true:false
+  const decreaseQunatity=(product)=>{  
+    setQuantityPrice((prevPrice)=>prevPrice-Number(product.price))
+    setCount((prevCount)=>prevCount-1)
+  } 
+  const increaseQuantity=(product)=>{
+    setQuantityPrice((prevPrice)=>prevPrice+Number(product.price))
+    setCount((prevCount)=>prevCount+1)
+  }
+
+  const removeProduct=async (product)=>{
+    try{
+      const updatedCartResponse=await axios.delete(`/api/user/cart/${product._id}`,{headers:{authorization:"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI3OGMwYThhMy1jN2ZiLTQ0ZjgtYWUwZS1iMmU2MTM2ZjUyNzEiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.RuUtRShcJPrhxsDlO30czFOKxmunlJd62KWyLPPwZlk"}})
+      dispatchCart({type:"cart",payload:{cart:updatedCartResponse.data.cart,cartCount:cartCount-1}})
+    }catch(e){
+      throw e
+    }   
+  }
+
+  const addToWishList=async(product)=>{
+    try{
+      await removeProduct(product)
+    }catch(e){
+      throw e
```

Comment:
why have you used throw here? when you can use console.log(e) here, just a question because i don't know what throw does.